pkgs/development/python-modules/backgroundremover/default.nix

```diff
@@ -0,0 +1,78 @@
+{ buildPythonPackage
+, certifi
+, charset-normalizer
+, fetchPypi
+, ffmpeg-python
+, filelock
+, filetype
+, gdown
+, hsh
+, idna
+, lib
+, more-itertools
+, moviepy
+, numpy
+, pillow
+, pymatting
+, pysocks
+, requests
+, scikitimage
+, scipy
+, six
+, torch
+, torchvision
+, tqdm
+, urllib3
+, waitress
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "backgroundremover";
+
+  version = "0.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-YAiPwI7YCTIDJxgzEoVCT/6htRemcVuvfj2lVS3dlks=";
+  };
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    ffmpeg-python
+    filelock
+    filetype
+    gdown
+    hsh
+    idna
+    more-itertools
+    moviepy
+    numpy
+    pillow
+    pymatting
+    pysocks
+    requests
+    scikitimage
+    scipy
+    six
+    torch
+    torchvision
+    tqdm
+    urllib3
+    waitress
+    flask
+  ];
+
+  doCheck = false; # no tests
```

Comment:
Can we enable `pythonImportsCheck` for a simple test?

Reply:
Thank you for reminding me. I forgot this part.

Reply:
done